### PR TITLE
Add full taxonomy to prototype

### PR DESCRIPTION
**Pull request by mobaig**
The prototype currently only pulls in taxons that are tagged to the content list we specify. This change adds a CSV list containing the complete education taxonomy, and uses it to complete the list of taxons that are processed in order to produce taxonomy_data.json.

There are several refactor/cleanup changes just prior to this - best reviewed by commit.

### CODE REVIEW

**Comment by MatMoore** [comment]
What's the reason for including the missing taxons? 

The original intent there was to avoid showing users links to topic pages that had no content tagged to them (or any of their children).

**Comment by mobaig** [comment]
@MatMoore to see what the design looks like with the full taxonomy, without needing to go through and tag content to every taxon. Looking to discuss with @alext how we intend to handle empty pages.

**Comment by alext** [comment]
@mobaig I'm not sure I have any context on this. I suspect you meant @alextea...

**Comment by mobaig** [comment]
Yep, sorry @alext !

File: lib/ruby/data_import.rb [hunk 1]
```diff
@@ -13,11 +13,14 @@ def self.rummager
...
+    unless base_path =~ %r{^\/}
+      base_path = "/" + base_path
+    end
     GdsApi.with_retries(maximum_number_of_attempts: 2) do
       true_base_path = find_redirects(base_path)
-      puts "Processing #{base_path}"
+      puts "Fetching #{base_path} from content store"
       puts "(Redirected to #{true_base_path})" if true_base_path != base_path
-      content_store.content_item!("/#{true_base_path}")
+      content_store.content_item!("#{true_base_path}")
```
Comment (carvil) [inline lib/ruby/data_import.rb]: There is no need for string interpolation anymore, we could write:

```
content_store.content_item!(true_base_path)
```

File: lib/ruby/data_import.rb [hunk 2]
```diff
@@ -27,10 +30,10 @@ def self.get_document(base_path)
   def self.find_redirects(base_path)
     http = Net::HTTP.new('www-origin.staging.publishing.service.gov.uk', 443)
     http.use_ssl = true
-    response = http.request_head("/#{base_path}")
+    response = http.request_head("#{base_path}")
```
Comment (carvil) [inline lib/ruby/data_import.rb]: Same as above, we could remove string interpolation.

**Pull request merged**